src/Sylius/Bundle/AdminBundle/Controller/NotificationController.php

```diff
@@ -81,6 +81,8 @@ public function getVersionAction(Request $request)
             return JsonResponse::create('', JsonResponse::HTTP_NO_CONTENT);
         }
 
-        return JsonResponse::fromJsonString($hubResponse->getBody()->getContents());
+        $hubResponse = json_decode($hubResponse->getBody()->getContents(), true);
+
+        return new JsonResponse($hubResponse);
```

Comment:
JsonResponse::fromJsonString does not work with ajax success function. I had not enough time to look deeper into what causes this problem, that's why I just left the new JsonResponse instance call. Looks like the fromJsonString was not used correctly.